src/main/java/com/suse/saltstack/netapi/calls/modules/Pkg.java

```diff
@@ -188,4 +188,32 @@ private Pkg() { }
         return new LocalCall<>("pkg.info_available", Optional.of(Arrays.asList(packages)),
                 Optional.empty(), new TypeToken<Map<String, Info>>(){});
     }
+
+    public static LocalCall<Boolean> upgradeAvailable(String packageName) {
+        LinkedHashMap<String, Object> args = new LinkedHashMap<>();
+        args.put("package", packageName);
+        return new LocalCall<>("pkg.upgrade_available", Optional.empty(), Optional.of(args),
+                new TypeToken<Boolean>(){});
+    }
+
+    public static LocalCall<String> latestVersion(String packageName) {
+        LinkedHashMap<String, Object> args = new LinkedHashMap<>();
+        args.put("package", packageName);
+        return new LocalCall<>("pkg.latest_version", Optional.empty(), Optional.of(args),
+                new TypeToken<String>(){});
+    }
+
+    public static LocalCall<Map<String, String>> latestVersions(String... packages)
```

Comment:
Lets avoid the exception by making the requirements explicit and take at least 2 package names.

```java
latestVersions(String first, String second, String... packages)
```

this function can be only called with a minimum 2 package names.